UnitsNet/QuantityFormatter.cs

```diff
@@ -13,6 +13,37 @@ namespace UnitsNet
     /// </summary>
     public class QuantityFormatter
     {
+        /// <summary>
+        /// The available UnitsNet custom format specifiers.
+        /// </summary>
+        private static readonly char[] UnitsNetFormatSpecifiers = { 'A', 'a', 'G', 'g', 'Q', 'q', 'S', 's', 'U', 'u', 'V', 'v' };
+
+        /// <summary>
+        /// Formats the given quantity using the given format string. Uses the <see cref="CultureInfo.CurrentUICulture" />.
```

Comment:
CurrentUICulture or CurrentCulture?

The former would be consistent with the current implementation, the latter would be more correct for the number formatting as discussed in #795.

Reply:
Yea, for consistency